lib/docker-sync/sync_strategy/rsync.rb

```diff
@@ -97,7 +97,7 @@ def start_container
             user_mapping = get_user_mapping
             group_mapping = get_group_mapping
 
-            cmd = "docker run -p '#{@options['sync_host_port']}:873' -v #{volume_name}:#{@options['dest']} #{user_mapping} #{group_mapping} -e VOLUME=#{@options['dest']} -e TZ=${TZ-`readlink /etc/localtime | sed -e 's,/usr/share/zoneinfo/,,'`} --name #{container_name} -d #{@docker_image}"
+            cmd = "docker run -p '#{@options['sync_host_port']}:873' -v #{volume_name}:#{@options['dest']} #{user_mapping} #{group_mapping} -e VOLUME=#{@options['dest']} -e TZ=${TZ-`readlink /etc/localtime | sed -e 's,/usr/share/zoneinfo/,,'`} -e ALLOW=#{@options['sync_host_allow']} --name #{container_name} -d #{@docker_image}"
```

Comment:
The `ALLOW` rule is to let it override the value here [entrypoint.sh#L86](https://github.com/EugenMayer/docker-image-rsyncd/blob/master/entrypoint.sh#L86), I came across this need because `Virtualbox` will use IPs in the `10.0.0.0/8` net when using the host-only network interface.

Some options to improve it:
1. Revert the change in this file and propose a PR against https://github.com/EugenMayer/docker-image-rsyncd to improve the IP range to support the `10.0.0.0/8` private network
2. Keep the override but let it be included only if the option was defined

Reply:
well i see what you mean, but then add the docs for this option sync_host_allow
here https://github.com/EugenMayer/docker-sync/blob/master/example/docker-sync.yml
and here https://github.com/EugenMayer/docker-sync/wiki/2.-Configuration
so people know about it, be sure to add a comment what this is for. 

thanks

Reply:
Don't we need default value for it? What if it's missing from the config?

Reply:
we cannot "guess" what private network there might be - so there cannot be a default value, right?

Reply:
but, without any default, `-e ALLOW=#{@options['sync_host_allow']} ` will turn into `-e ALLOW=` which I assume is not a valid syntax?

if we can't have default, then the argument should be added  only when `@options['sync_host_allow']` is not blank?

Reply:
sure that will be needed, right - so if blanc, skip the whole argument

Reply:
If value is blank, then bash will default to `192.168.0.0/16 172.16.0.0/12` since the rule is constructed like: `ALLOW=${ALLOW:-192.168.0.0/16 172.16.0.0/12}`, still I do agree that's cleaner to just avoid the parameter if unset.

Reply:
@EugenMayer, @ignatiusreza, changes done